waku/v2/protocol/waku_swap/waku_swap.nim

```diff
@@ -276,4 +288,14 @@ proc setPeer*(ws: WakuSwap, peer: PeerInfo) =
   ws.peerManager.addPeer(peer, WakuSwapCodec)
   waku_swap_peers_count.inc()
 
+proc getPeerConnectionStatus*(ws: WakuSwap, peerId : PeerId) : PeerConnectionStatus = 
+  result.balance = 0;
```

Comment:
```suggestion
  result.balance = 0
```

Reply:
we avoid using result returns https://status-im.github.io/nim-style-guide/language.result.html